commonLibrary/src/main/java/com/skytask/common/ProductMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.skytask.common;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.CollectionType;
+
+import java.io.IOException;
+import java.util.List;
+
+public class ProductMapper {
+    public List<Product> json2ProductList(String json) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
You're creating  ObjectMapper instance in every method you use it. can you figure out a way to avoid having repeated the same lines of code?

Reply:
I am thinking in declaring an object mapper as avariable, to use it in both methods, without making and instance in each one of them.